hbase_storage/src/main/java/com/splicemachine/storage/HLock.java

```diff
@@ -28,13 +28,11 @@
  *         Date: 12/15/15
  */
 public class HLock implements Lock{
-    protected static boolean HAS_READ_LOCK_IMPL = false;
 
     static {
```

Comment:
Remove the whole static initialization block, it's not used for anything other than HAS_READ_LOCK_IMPL, right?